_includes/configuration/advtable-value-series.md

```diff
@@ -0,0 +1,82 @@
+## `advtable_value_series`
+{{ site.requires_5_8 }}
+
+{% if pluginname != "Advanced Tables" %}
+> **Note**: The `advtable_value_series` option requires the Advanced Tables plugin.
+{% endif %}
+
+The `advtable_value_series` option allows values series to be configured that can be used to populate cells in a table.
+
+**Type:** `Object`
+
+**Default Value:**
+ - `numbers` - a natural number series.
+ - `alpha` - a base26 letter series.
+
+```js
+{
+  numbers: {
+    update: true,
+    resizable: false,
+    generator: ...
```

Comment:
will this take a function?

Reply:
Do we need to write the function out here, or is it a common pattern in the docs to put `...` into code snippets?

Reply:
According to the [style guide](https://www.tiny.cloud/docs/configure/contributing-docs/#codesnippets), `...` in code samples is bad and shouldn't be used.

Reply:
yeah, please provide a full example. Ideally, users should be able to copy+paste codeblocks into, say, fiddle and have the code work without modification (excluding having to change the selector).

Reply:
Have removed the `...`